modules/ROOT/pages/update-streams-switching.adoc

```diff
@@ -0,0 +1,10 @@
+
+= Switching to a Different Stream
```

Comment:
While it should work most of the time, we should probably have a warning in this section that things might break due to skipping update barriers. We can add that as a follow-up though.

Reply:
Related concern, manual rebases may result in downgrades, which basically no OS component is prepared to handle gracefully.
We'd better have a blurb here stating which stream-switching direction is sane.

Reply:
Added a note!

Reply:
added a note about "downgrades"